packages/shared/lib/participation/staking.ts

```diff
@@ -370,7 +379,107 @@ export const getTimeUntilMinimumAirdropReward = (account: WalletAccount, format:
     if (!account) return format ? getBestTimeDuration(0) : 0
 
     const [minRewards, minAirdrop, amountStaked] = getMinimumAirdropRewardInfo(account)
```

Comment:
Using the current implementation of getMinimumAirdropRewardInfo, this will return [0, undefined, 0] which will intern cause this function to return 0. This is because you inside of getMinimumAirdropRewardInfo you only get the information if the account has or is participating in an airdrop.